crates/core/src/network/handler/lists.rs

```diff
@@ -57,7 +115,9 @@ fn create_authoring_map(context: Arc<Context>) -> HashMap<EntryHash, Vec<AspectH
             &Vec::new(),
         ).expect("Must be able to create dummy header header when responding to HandleGetAuthoringEntryList");
         let content_aspect = EntryAspect::Content(chain_header_entry, header_entry_header);
-        address_map.insert(entry_hash, vec![AspectHash::from(content_aspect.address())]);
+        let mut set = HashSet::new();
+        set.insert(AspectHash::from(content_aspect.address()));
+        address_map.insert(entry_hash, set);
```

Comment:
Is there a chance that this loop could visit the same entry_hash multiple times? If so then we will overwrite the aspect set for that entry on each loop..